src/pages/mypage/PostsIdeaMarket/postsIdeaMarket.module.scss

```
@@ -0,0 +1,85 @@
+.postcard-details {
+  padding: 20px;
+  font-family: Arial, sans-serif;
+
+  .postcard-header {
+    display: flex;
+    align-items: center;
+    margin-bottom: 20px;
+
+    .image-placeholder {
+      width: 100px;
+      height: 100px;
+      background-color: #f0f0f0;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      margin-right: 20px;
+    }
+
+    .postcard-info {
+      h1 {
+        font-size: 1.5rem;
+        margin: 0;
+      }
+
+      p {
+        font-size: 1.2rem;
+        color: #555;
+      }
+    }
+  }
+
```

2 단계 이상으로 Depth가 깊어지면 가독성이 떨어질 수 있어서 가급적 1단계까지 깊어지는 것은 추천드립니동! 저스트 저의 의견...

```suggestion
  .postcard-header {
    display: flex;
    align-items: center;
    margin-bottom: 20px;

    .image-placeholder {
      width: 100px;
      height: 100px;
      background-color: #f0f0f0;
      display: flex;
      justify-content: center;
      align-items: center;
      margin-right: 20px;
    }
}
 .postcard-info {
      h1 {
        font-size: 1.5rem;
        margin: 0;
      }

      p {
        font-size: 1.2rem;
        color: #555;
      }
  }

```

---

넵 코드 리뷰 해주신 부분 참고해서 수정하도록 하겠습니다 감사합니다~!